### PR TITLE
Cp/home page

### LINKED ISSUE #22: [Pages] [due June 20] Homepage
# Context

Create the homepage using the components library as per the design [here](https://www.figma.com/file/704SXr4HrUbIIkXYena62Q/Flow-Dev-Portal-(External)?node-id=1012%3A35901).

### PR DESCRIPTION
**Pull request by aureentuluvacal**
Closes #22

### CODE REVIEW

**Review by joekrill — APPROVED** [review]

File: libs/design-system/src/lib/Pages/HomePage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    <div className="bg-primary-gray-50 dark:bg-black">
+      <LandingHeaderHome
+        title="Developer Resources"
+        description="Understand the foundational concepts of Flow and its language, Cadence"
+        tag="onflow"
+      />
+
+      <LinkCard2Column {...startProjectItems} bottomRounded={false} />
+
+      <div className="mb-36">
```
Comment (alse) [inline libs/design-system/src/lib/Pages/HomePage/index.tsx:42]: The new PageSections and PageSection components could be used here for spacing and horizontal borders

File: libs/design-system/src/lib/Components/ToolsAndConcepts/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { ToolCard, ToolCardProps } from '../ToolCard';
+import { ButtonLink } from '../Button';
+import { ReactComponent as RightChevronIcon } from '../../../../images/arrows/chevron-right.svg';
+import { ReactComponent as ExternalLinkIcon } from '../../../../images/content/external-link.svg';
+
+export type ToolsAndConceptsProps = {
+  tools: ToolCardProps[];
+};
+
+const ToolsAndConcepts = ({ tools }) => (
```
Comment (alse) [inline libs/design-system/src/lib/Components/ToolsAndConcepts/index.tsx:11]: These may conflict with #106

File: libs/design-system/src/lib/Pages/HomePage/index.tsx (hunk 1 above)
Comment (aureentuluvacal) [reply libs/design-system/src/lib/Pages/HomePage/index.tsx:42]: Doesn't look like there's borders on the home page or am I crazy?

File: libs/design-system/src/lib/Components/ToolsAndConcepts/index.tsx (hunk 2 above)
Comment (aureentuluvacal) [reply libs/design-system/src/lib/Components/ToolsAndConcepts/index.tsx:11]: Indeed. Fixing it now.

File: libs/design-system/src/lib/Pages/HomePage/HomePage.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,190 @@
+import { Meta, Story } from '@storybook/react';
+import HomePage, { HomePageProps } from '.';
+import { ReactComponent as UseCaseIcon } from '../../../../images/content/use-cases.svg';
+import { ReactComponent as EcosystemIcon } from '../../../../images/content/ecosystem.svg';
+import { ReactComponent as SDKIcon } from '../../../../images/content/sdk.svg';
+import KittyItemsImg from '../../../../images/content/kitty-items@3x.png';
+import DappyImg from '../../../../images/content/dappy@3x.png';
```
Comment (alse) [inline libs/design-system/src/lib/Pages/HomePage/HomePage.stories.tsx:7]: Should we keep the placeholder images in the current story folder?

**Pull request merged**